SETUP.md

```diff
@@ -95,9 +95,28 @@ We recommend using either GitHub Actions or Travis CI, but not both to avoid dep
 AppVeyor can be used in addition to GitHub Actions or Travis CI to comment on pull request with download links to rendered PDFs.
 GitHub Actions do upload rendered manuscripts as artifacts, but do not leave pull request comments.
 
+### GitHub Actions
+
+If you plan on only using GitHub actions, you can remove configuration files for other CI services:
+
+```shell
+# remove Travis CI config
+git rm .travis.yml
+# remove AppVeyor config
+git rm .appveyor.yml
+# remove ci/install.sh if not using neither Travis CI nor AppVeyor
+git rm ci/install.sh
+```
+
+<details>
+<summary>Expand for Deploy Key</summary>
+
 ### Deploy key
 
-Generate a deploy key so CI can write to the repository.
+Deployment on Travis CI requires a SSH Deploy Key.
+Previousely, GitHub Actions also required an SSH Deploy Key, but now GitHub supports using `secrets.GITHUB_TOKEN`.
```

Comment:
`Previousely` -> `Previously`

Do we want to explain the differences between GITHUB_TOKEN, an SSH deploy key, and a GitHub personal access token in terms of permissions?  These terms can be confusing for people who aren't experienced with continuous integration.  Others may mix up this GITHUB_TOKEN with the broader personal access token.

Reply:
I updated the SETUP extensively, hopefully a bit clearer on these points although still don't go into too much detail.